app/frontend/package.json

```diff
@@ -35,16 +35,20 @@
     "bootstrap-scss": "^4.6.1",
     "core-js": "^3.23.3",
     "deep-freeze": "^0.0.1",
+    "export-from-json": "^1.7.2",
     "fast-json-patch": "^3.1.1",
     "font-awesome": "^4.7.0",
+    "formio-utils": "^0.2.4",
     "formiojs": "^4.14.6",
     "keycloak-js": "^15.0.2",
     "material-design-icons-iconfont": "^6.7.0",
     "moment": "^2.29.3",
     "nprogress": "^0.2.0",
+    "papaparse": "^5.4.0",
     "qrcode.vue": ">= 1.7.0 < 3.0.0",
     "uuid": "^8.3.2",
     "vue": "^2.7.2",
+    "vue-blob-json-csv": "^0.1.7",
```

Comment:
What is the purpose of this package "vue-blob-json-csv": "^0.1.7"

Reply:
This package is used to convert json to CSV file , we use it in the app to generate report to the submitter in a csv file .